pilot/pkg/security/authz/builder/testdata/http/allow-path-out.yaml

```diff
@@ -22,6 +22,16 @@ typedConfig:
                     path:
                       safeRegex:
                         regex: .+
+                - uriTemplate:
+                    name: uri-template
+                    typedConfig:
+                      '@type': type.googleapis.com/envoy.extensions.path.match.uri_template.v3.UriTemplateMatchConfig
+                      pathTemplate: /path/template/*
+                - uriTemplate:
+                    name: uri-template
+                    typedConfig:
+                      '@type': type.googleapis.com/envoy.extensions.path.match.uri_template.v3.UriTemplateMatchConfig
+                      pathTemplate: '**/path/template'
```

Comment:
Why is this different from L60? no `/` prefix

Reply:
Typo - will fix

Reply:
how did it pass the test then?

Reply:
or was the input missing the prefix slash as well